nms/app/packages/magmalte/app/state/AlarmsApiUtil.js

```diff
@@ -39,11 +39,30 @@ export const MagmaAlarmsApiUtil: ApiUtil = {
     console.warn('not implemented');
     return [];
   },
-  getTroubleshootingLink: async ({alertName: _}) =>
-    Promise.resolve({
-      link: '',
+  getTroubleshootingLink: async ({alertName}) => {
+    const alertLinks = [
```

Comment:
this list is independent of the api, lets pass it in as an arg instead

Reply:
Sorry, could you elaborate a bit on this point?